usaspending_api/references/tests/test_financial_spending.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import json
+from datetime import date
+
+from model_mommy import mommy
+from rest_framework import status
+
+from usaspending_api.awards.models import Award
+from usaspending_api.references.models import Agency, ToptierAgency, SubtierAgency
+
+
+@pytest.fixture
+def financial_spending_data(db):
+    # agency w/ tt agency
+    ttagency1 = mommy.make('references.ToptierAgency', cgac_code="cgac_code", name="tta_name")
+    mommy.make('references.Agency', id=1, toptier_agency=ttagency1)
+    # TAS w/ same ttagency
+    tas1 = mommy.make('accounts.TreasuryAppropriationAccount', awarding_toptier_agency=ttagency1, funding_toptier_agency=ttagency1)
+
+    # Financial Account with Object class and submission
+    object_class_1 = mommy.make('references.ObjectClass', major_object_class="mocCode",
+                                major_object_class_name="mocName", object_class="ocCode", object_class_name="ocName")
+    submission_1 = mommy.make('submissions.SubmissionAttributes', reporting_fiscal_year=2017, cgac_code="cgac_code")
```

Comment:
What do you think about not explicitly setting`cgac_code` on the submission? 

Thinking from the perspective of a new dev who is learning the code base by reading the tests....setting `cgac_code="cgac_code"` implies that it's significant to the API lookup, when in reality the only toptier agency info that matters is the `funding_toptier_agency` that you set in `tas1`.